docs/faq/events.md

```diff
@@ -28,3 +30,182 @@ There are lots of times when words just aren't enough to really get clarity on a
     style="height: 100%; width: 100%; border: 0;">
   </iframe>
 </div>
+
+## Event listening
+
+When preparing to listen for events in your code, do the least work possible. Not only because this allows you to do less work now, but it allows you also to do less work later, and most importantly it allows the browser to do less work, which means a more performant application or component. The leasst work possible changes by context, but in important to remember no matter where you are adding event listners.
```

Comment:
```suggestion
When preparing to listen for events in your code, do the least work possible. Not only because this allows you to do less work now, but it allows you also to do less work later, and most importantly it allows the browser to do less work, which means a more performant application or component. The least work possible changes per context, but in important to remember no matter where you are adding event listeners.
```